primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -2166,6 +2166,8 @@
                     },
                     onEntered: function() {
                         if (!$this.options.touchUI) {
+                            $this.datepickerClick = true;
+                            setTimeout(function () {$this.datepickerClick = false;}, 200);
```

Comment:
I guess you probably already looked at a better way to fix this?

Reply:
I have dug around and this is the best I could come up with since it only happens strangely with a mousepad and its a timing issue related to the animation etc.  There are so many complex events going on that this was the only thing I can do to get it working without breaking other things!